pkg/internal/templates/cloud-init.sh.template

```diff
@@ -44,6 +44,15 @@ mkdir -p '{{ $unit.DropIns.Path }}'
 {{- end -}}
 {{- end }}
 
+# Change existing worker to use docker registry-mirror
+file="/etc/docker/daemon.json"
+if [ $(jq -r 'has("registry-mirrors")' "${file}") == "false" ]; then
+    contents=$(jq -M '. += {"registry-mirrors": ["https://mirror.gcr.io"]}' ${file})
+    echo "${contents}" > ${file}
+    systemctl daemon-reload
```

Comment:
i dont think damon-reload is required because no systemd unit file was changed.

Reply:
If it's the first time this is executed (`.Bootstrap` == true), the docker.service unit file was touched beforehand (https://github.com/metal-stack/os-metal-extension/blob/b457e4090d272b28158ee35f32e6f60eb00d0e53/pkg/internal/templates/cloud-init.sh.template#L24), so there it is necessary

Reply:
OK, unsure but dont hurt.